src/sentry/ingest/transaction_clusterer/__init__.py

```diff
@@ -1 +1,26 @@
 """ Strategies for clustering high-cardinality transaction names """
+
+
+from enum import Enum
+from typing import Dict
+
+
+class ClustererDataNamespace(Enum):
+    TRANSACTIONS = "txnames"
+
+
+class ClustererRuleNamespace(Enum):
+    TRANSACTIONS = "txrules"
+
+
+class PROJECT_OPTION_KEYS(Enum):
+    STORAGE = "storage_option"
+    TRACKER = "tracker_option"
+
+
+PROJECT_OPTION_NAMES: Dict[ClustererRuleNamespace, Dict[PROJECT_OPTION_KEYS, str]] = {
+    ClustererRuleNamespace.TRANSACTIONS: {
+        PROJECT_OPTION_KEYS.STORAGE: "sentry:transaction_name_cluster_rules",
+        PROJECT_OPTION_KEYS.TRACKER: "txcluster.rules_per_project",
+    }
+}
```

Comment:
In total, we need 2 x 4 = 8 strings, as seen in this table:

| type             | rule | name | storage | tracker |
| ---------------- | ---- | ---- | ------- | ------- |
| TRANSACTION      |      |      |         |         |
| SPAN_DESCRIPTION |      |      |         |         |

`ClusterDataNamespace` and `ClusterRuleNamespace` each represent a _column_ in this table, but `PROJECT_OPTION_KEYS` represents a (partial) _row_ of this table. Instead of this mixture I would chose either rows or columns consistently.